packages/flutter/test/material/button_bar_test.dart

```diff
@@ -95,7 +95,7 @@ void main() {
       );
 
       // ButtonBar uses a Row internally to implement this
```

Comment:
It now uses a `ButtonBarRow` instead of a `Row` now. :)

Reply:
I was just trying to write tests for `ButtonBarRow` and `MainAxisAlignment` and cannot seem to find how this parameter is used. As far as I understand, it seems to only be valuable when Flexible children are present, but since each button is wrapped in a Padding widget, there is no way to make a child of ButtonBar flexible. I tried playing around with the param in a small app and see no difference between setting it to min and max.

Deleting the parameter would be a breaking change for apps that define this param, so maybe we could delete the tests for now, and then in a follow-up PR, try removing the parameter and see if anything breaks?

Reply:
Nevermind - I misunderstood how MainAxisSize works. It turns out that this basically tells its own parent how much space it intends to take. However, this just means that if ButtonBar.mainAxisSize was set to MainAxisSize.min, it sort of ignores alignment since there's minimal space and is aligned based on how the parent itself chooses to align itself.